How to add a custom css file

I tried the following Makefile
```
NAME = README
HTML_CSS = styles/doc.css

include makespec/Makefile
```
(The doc suggests HTML_TEMPLATE as variable name here, but I suppose that's a typo).

Anyway, I get invariably:

    make: *** No rule to make target `styles/doc.css', needed by `README.html'.  Stop.

--- a/formats.make
+++ b/formats.make
@@ -43,7 +43,7 @@ ifneq ($(HTML_CSS),)
     HTML_ARGS += --css=$(HTML_CSS) 
 endif
 
-$(NAME).html: $(COMBINED) status
+$(NAME).html: $(COMBINED) $(HTML_TEMPLATE) status
 	@echo "Creating $@..."
 	@$(PANDOC) $(HTML_ARGS) -f markdown -t html5 $(VARS) $(HTML_VARS) $(COMBINED) \
 		| perl -p -e 's!(http://[^<]+)\.</p>!<a href="$$1"><code class="url">$$1</code></a>.</p>!g' \

--- a/formats.make
+++ b/formats.make
@@ -43,7 +43,7 @@ ifneq ($(HTML_CSS),)
     HTML_ARGS += --css=$(HTML_CSS) 
 endif
 
-$(NAME).html: $(COMBINED) $(HTML_TEMPLATE) $(HTML_CSS) status
+$(NAME).html: $(COMBINED) status
 	@echo "Creating $@..."
 	@$(PANDOC) $(HTML_ARGS) -f markdown -t html5 $(VARS) $(HTML_VARS) $(COMBINED) \
 		| perl -p -e 's!(http://[^<]+)\.</p>!<a href="$$1"><code class="url">$$1</code></a>.</p>!g' \

